### PR TITLE
docs: minor fixes to contribution guidelines

### PR DESCRIPTION
**Pull request by Zero-1729**
This PR only fixes a minor typo and phrasing.

### CODE REVIEW

File: docs/code_contribution_guidelines.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Developing cryptocurrencies is an exciting endeavor that touches a wide variety
 of areas such as wire protocols, peer-to-peer networking, databases,
 cryptography, language interpretation (transaction scripts), adversarial
 threat-modeling, and RPC systems. They also represent a radical shift to the
-current fiscal system and as a result provide an opportunity to help reshape
+current financial system and as a result provide an opportunity to help reshape
```
Comment (yyforyongyu) [inline docs/code_contribution_guidelines.md:30]: why the change?

Comment (Zero-1729) [reply docs/code_contribution_guidelines.md:30]: I wasn't sure how LN is related to fiscal systems (gov revenue & expenditure, taxes, etc.)? Now on second thought, rewording it to financial makes it tautological.

So I am thinking `monetary` system might be a better rewording as Bitcoin **is** a radically different monetary network/system vis-a-vis the existing one.

Comment (Zero-1729) [reply docs/code_contribution_guidelines.md]: @yyforyongyu Let me know what you think, I just swapped out `fiscal` for `monetary` in f1ac44dd01615370eb44388f750f1c70d77e5ed0 instead of `financial` for the reasons I mentioned above.

Comment (yyforyongyu) [reply docs/code_contribution_guidelines.md]: IMO, the gov will be forced to accept LN and thus one day build its fiscal policy on top of LN, mostly taxation. `monetary` is more about managing the supply/demand of money. To me it's like two different ways for the gov to manage the financial system, and I'm ok with either word.

Strictly speaking, I would not consider this a typo fix as it's related to semantics. However since it's not related to code change so I think it's ok.

**Review by yyforyongyu — APPROVED** [review]
LGTM

File: docs/code_contribution_guidelines.md (hunk 1 above)
Comment (Zero-1729) [reply docs/code_contribution_guidelines.md]: Ah, fair enough. Thanks for clarifying.

I'll update the commit message to something more general to reflect your point about it not being a typo. This way, it removes any confusion in the codebase down the line 👍🏽

**Review by carlaKC — APPROVED** [review]

**Comment by guggero** [comment]
We added a new check that makes sure every PR has a short (one sentence) description in the release notes. If you rebase, you'll find a section `Code health -> Code cleanup, refactor, typo fixes`. Can you add the link to your PR there please?

**Comment by Zero-1729** [comment]
@guggero I also added bullet points to the contributors names, so it doesn't appear cluttered as it does in the [v0.13.1 release notes](https://github.com/lightningnetwork/lnd/blob/master/docs/release-notes/release-notes-0.13.1.md). However, if you feel it unnecessarily breaks precedent, I'll be happy to revert the change to match the current [v0.13.1 contributors listing format](https://github.com/lightningnetwork/lnd/blob/master/docs/release-notes/release-notes-0.13.1.md#contributors-alphabetical-order).
